README.md

```diff
@@ -139,10 +139,11 @@ Variables that you want to make available to other package are in the `export` b
 In the example above, the `OUTPUT` variable is created as part of a Zarf Action in the [output-var](src/test/packages/zarf/no-cluster/output-var) package, and the [receive-var](src/test/packages/zarf/no-cluster/receive-var) package expects a variable called `OUTPUT`.
 
 ### Variable Precedence and Specificity
-In a bundle, variables can come from 3 sources. Those sources and their precedence are shown below in order of least to most specificity:
+In a bundle, variables can come from 4 sources. Those sources and their precedence are shown below in order of least to most specificity:
 - Variables declared in a Zarf pkg
 - Variables `import`'ed from a bundle package's `export`
-- Variables declared in `uds-config.yaml`
+- Variables configured in `uds-config.yaml`
+- Variables set with an environment variable prefixed with `UDS_` (ex. `UDS_OUTPUT`)
 
 That is to say, deploy-time variables declared in `uds-config.yaml` take precedence over all other variable sources.
```

Comment:
would env variables with `UDS_` prefix now take precedence?

Reply:
For sure! The list goes from least specific to most specific, so `UDS_` env vars take precedence over anything else